exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/metric_exporter/__init__.py

```diff
@@ -368,93 +177,6 @@ def force_flush(self, timeout_millis: float = 10_000) -> bool:
         return True
 
 
-def _translate_value(value: Any) -> KeyValue:
-
-    if isinstance(value, bool):
-        any_value = AnyValue(bool_value=value)
-
-    elif isinstance(value, str):
-        any_value = AnyValue(string_value=value)
-
-    elif isinstance(value, int):
-        any_value = AnyValue(int_value=value)
-
-    elif isinstance(value, float):
-        any_value = AnyValue(double_value=value)
-
-    elif isinstance(value, Sequence):
-        any_value = AnyValue(
-            array_value=ArrayValue(values=[_translate_value(v) for v in value])
-        )
-
-    elif isinstance(value, Mapping):
-        any_value = AnyValue(
-            kvlist_value=KeyValueList(
-                values=[
-                    _translate_key_values(str(k), v) for k, v in value.items()
-                ]
-            )
-        )
-
-    else:
-        raise Exception(f"Invalid type {type(value)} of value {value}")
-
-    return any_value
-
-
-def _translate_key_values(key: str, value: Any) -> KeyValue:
-    return KeyValue(key=key, value=_translate_value(value))
-
-
-def get_resource_data(
```

Comment:
The list of symbols that are currently publicly accessible to users which are getting removed in this PR as a part of refactoring work.
- get_resource_data
- DEFAULT_COMPRESSION
- DEFAULT_ENDPOINT
- DEFAULT_TIMEOUT

And there are a bunch of transitive imports symbols getting removed as well. This is technically a breaking change since one could have something like following.

```py
from opentelemetry.exporter.otlp.proto.http.metric_exporter import get_resource_data, DEFAULT_ENDPOINT, DEFAULT_TIMEOUT
```

We shouldn't introduce any breaking changes simultaneously, but we still want to refactor as much as possible. @open-telemetry/python-approvers what are your thought on this.

Reply:
IMO we should not break users so we should probably leave these symbols there. Maybe include a comment about how these are deprecated?

Reply:
yeah, I'll update it.